twilight-cache-inmemory/src/event/guild.rs

```diff
@@ -432,6 +434,7 @@ mod tests {
             public_updates_channel_id: None,
             roles: Vec::new(),
             rules_channel_id: None,
+            safety_alerts_channel_id: None,
```

Comment:
Maybe this could be changed to `Some` to ensure it's cached, but neither are other fields so maybe we don't test these? If so, why?